src/test/java/net/sf/jabref/logic/importer/fetcher/MedlineFetcherTest.java

```diff
@@ -0,0 +1,182 @@
+        entryWijedasa.setField("copyright", "This article is protected by copyright. All rights reserved.");
+        entryWijedasa.setField("created", "2016-9-27");
+        entryWijedasa.setField("doi", "10.1111/gcb.13516");
+        entryWijedasa.setField("issn", "1365-2486");
+        entryWijedasa.setField("issn-linking", "1354-1013");
+        entryWijedasa.setField("journal", "Global change biology");
+        entryWijedasa.setField("journal-abbreviation", "Glob Chang Biol");
+        entryWijedasa.setField("keywords", "Acacia; Agriculture; Emissions; Subsidence; Sustainability; Tropical peatlands; oil palm");
+        entryWijedasa.setField("month", "Sep");
+        entryWijedasa.setField("nlm-id", "9888746");
+        entryWijedasa.setField("owner", "NLM");
+        entryWijedasa.setField("pmid", "27670948");
+        entryWijedasa.setField("pubmodel", "Print-Electronic");
+        entryWijedasa.setField("pubstatus", "aheadofprint");
+        entryWijedasa.setField("revised", "2016-9-27");
+        entryWijedasa.setField("status", "Publisher");
+        entryWijedasa.setField("title", "Denial of long-term issues with agriculture on tropical peatlands will have devastating consequences.");
+        entryWijedasa.setField("year", "2016");
+
+        entryEndharti = new BibEntry();
+        entryEndharti.setType(BibLatexEntryTypes.ARTICLE);
+        entryEndharti.setField("title", "Dendrophthoe pentandra (L.) Miq extract effectively inhibits inflammation, proliferation and induces p53 expression on colitis-associated colon cancer.");
+        entryEndharti.setField("author", "Endharti, Agustina Tri and Wulandari, Adisti and Listyana, Anik and Norahmawati, Eviana and Permana, Sofy");
+        entryEndharti.setField("created", "2016-9-27");
+        entryEndharti.setField("issn", "1472-6882");
+        entryEndharti.setField("issn-linking", "1472-6882");
+        entryEndharti.setField("issue", "1");
+        entryEndharti.setField("journal", "BMC complementary and alternative medicine");
+        entryEndharti.setField("journal-abbreviation", "BMC Complement Altern Med");
+        entryEndharti.setField("keywords", "CAC; Dendrophtoe pentandra; IL-22; MPO; Proliferation; p53");
+        entryEndharti.setField("nlm-id", "101088661");
+        entryEndharti.setField("owner", "NLM");
+        entryEndharti.setField("pages", "374");
+        entryEndharti.setField("pmid", "27670445");
+        entryEndharti.setField("pubmodel", "Electronic");
+        entryEndharti.setField("pubstatus", "epublish");
+        entryEndharti.setField("revised", "2016-9-27");
+        entryEndharti.setField("status", "Publisher");
```

Comment:
Do you know what this "status" field is? Looks to me like we can safely delete it.